LICENSE-binary

```diff
@@ -280,6 +280,11 @@ org.jline:jline:3.21.0
 com.github.luben:zstd-jni:1.5.4-2
 
 
+BSD 2-Clause
+------------
+com.github.luben:zstd-jni:1.5.4-2
```

Comment:
BSD 2-Clause + BSD 3-Clause ?

Reply:
BSD 2-Clause, I was wrong about that. The native Zstd is under 3-clause BSD licence. The JNI bindings is licenced under 2-clause BSD license.